src/injected/visualization/tab-stops-formatter.ts

```diff
@@ -58,6 +73,13 @@ export class TabStopsFormatter implements Formatter {
                 strokeWidth: '3',
                 strokeDasharray: '7 2',
             },
+            failureBoxConfig: {
```

Comment:
nit: the corners in the mock look slightly rounded. Could we add that here?

Reply:
Yup! Added that so the failure labels now look like this:
![RoundedCorner](https://user-images.githubusercontent.com/16010855/154366370-319ed50d-81f3-4545-9963-a4e7cf11f6c5.png)
(screenshot of updated failure label, red box with '!' in top right corner of tab stops circle, now with rounded corners)